client/src/FiltersPage.js

```diff
@@ -0,0 +1,181 @@
+
+export default function FiltersPage() {
+  ///////////////////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////////////////////////////////
+  // Map and List view mode buttons
+
+  const [mode, setMode] = React.useState(1);
+
+  function render() {
+    if (mode == 1) {
+      return (
+        <div>
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+          <CardComponent
+            name="Bianca's Radishes"
+            location="Berkeley, CA"
+            tags={['vegetables', 'fruit']}
+          />
+        </div>
+      );
+    } else {
+      return <div>map please </div>;
+    }
+  }
+
+  function list_button() {
+    if (mode == 1) {
+      return (
+        <button
+          className="activeButton"
+          type="button"
+          onClick={() => setMode(1)}
+        >
+          List
+        </button>
+      );
+    } else {
+      return (
+        <button
+          className="inactiveButton"
+          type="button"
+          onClick={() => setMode(1)}
+        >
+          List
+        </button>
+      );
+    }
+  }
+
+  function map_button() {
+    if (mode == 2) {
+      return (
+        <button
+          className="activeButton"
+          type="button"
+          onClick={() => setMode(2)}
+        >
+          Map
+        </button>
+      );
+    } else {
+      return (
+        <button
+          className="inactiveButton"
+          type="button"
+          onClick={() => setMode(2)}
+        >
+          Map
+        </button>
+      );
+    }
+  }
+
+  ///////////////////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////////////////////////////////
+  // Filter Dropdowns
+
+  const [role, setRole] = React.useState([]);
+  const [location, setLocation] = React.useState([]);
+  const [race, setRace] = React.useState([]);
+  const [products, setProducts] = React.useState([]);
+  const [other, setOther] = React.useState([]);
+  console.log(role, location, race, products, other);
```

Comment:
nit: change to `console.log({role, location, race, products, other})` so it'll be more clear which arrays correspond to which